API.md

```diff
@@ -14,20 +16,38 @@ An object for encapsulating an Amplify Geo transform request and Amplify credent
-*   `region` **[String][5]** AWS region
+*   `region` **[String][7]** AWS region
 
-Returns **[AmplifyMapLibreRequest][6]** `this`
+Returns **[AmplifyMapLibreRequest][8]** `this`
 
 ### transformRequest
 
-A callback function that can be passed to a maplibre map object that is run before the map makes a request for an external URL. This transform request is used to sign the request with AWS Sigv4 Auth. [https://maplibre.org/maplibre-gl-js-docs/api/map/][7]
+A callback function that can be passed to a maplibre map object that is run before the map makes a request for an external URL. This transform request is used to sign the request with AWS Sigv4 Auth. [https://maplibre.org/maplibre-gl-js-docs/api/map/][9]
 
 #### Parameters
 
-*   `url` **[string][5]** The function to use as a render function. This function accepts a single [Carmen GeoJSON][8] object as input and returns a string.
-*   `resourceType` **[string][5]** The function to use as a render function. This function accepts a single [Carmen GeoJSON][8] object as input and returns a string.
+*   `url` **[string][7]** The function to use as a render function. This function accepts a single [Carmen GeoJSON][10] object as input and returns a string.
+*   `resourceType` **[string][7]** The function to use as a render function. This function accepts a single [Carmen GeoJSON][10] object as input and returns a string.
 
-Returns **RequestParameters** [https://maplibre.org/maplibre-gl-js-docs/api/properties/#requestparameters][9]
+Returns **RequestParameters** [https://maplibre.org/maplibre-gl-js-docs/api/properties/#requestparameters][11]
+
+## drawPoints
+
+DrawPoints utility function for adding points to a map based on coordinate data or a FeatureCollection. Will add clustered points and styled markers by default with options for popups and other styles
+
+### Parameters
+
+*   `sourceName` **[String][7]** A user defined name used for determining the maplibre data source and the maplibre layers
+*   `data` **([Array][12]\<Coordinate> | [Array][12]\<Feature>)** An array of coordinate data or GeoJSON Features used as the data source for maplibre
+*   `map` **maplibreMap** A maplibre map on which the points will be drawn
+*   `options` **[Object][13]** An object containing options for changing the styles and features of the points rendered to the map, see the options for more details on available settings
+
+    *   `options.showCluster` **[String][7]** Determines whether or not points close together should be clustered into a single point (optional, default `true`)
+    *   `options.clusterOptions` **[String][7]** Object for determining cluster options, see [ClusterOptions][14] for more details (optional, default `{}`)
+    *   `options.unclusteredOptions` **[String][7]** Object for determining cluster options, see [UnclusteredOptions][15] for more details (optional, default `{}`)
+*   `mapStyle` **[String][7]** A string containing the map style data used to draw this map
```

Comment:
How is mapStyle used in the `drawPoints`?

Reply:
It is used for determining the default font set to use, I can add that to the comments.